### PR TITLE
Implement card 'Gadgetzan Auctioneer' (EX1_095) and 'Holy Fire' (EX1_624)

### PR DESCRIPTION
**Pull request by thrk941106**

### CODE REVIEW

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp [hunk 1]
```diff
@@ -170,6 +170,20 @@ void Expert1CardsGen::AddPriest(std::map<std::string, Power>& cards)
...
+    // [EX1_624] Holy Fire - COST:6
+    // - Faction: Priest, Set: Expert1, Rarity: Rare
+    // --------------------------------------------------------
+    // Text: Deal $5 damage. Restore #5 Health to your hero.
+    // --------------------------------------------------------
+    // PlayReq:
+    // - REQ_TARGET_TO_PLAY = 0
+    // --------------------------------------------------------
+    power.ClearData();
+    power.AddPowerTask(new DamageTask(EntityType::TARGET, 5, false));
```
Comment (utilForever) [inline Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: `Holy Fire` is spell damage. You should set third parameter to `true`.

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp [hunk 2]
```diff
@@ -894,6 +908,19 @@ void Expert1CardsGen::AddNeutral(std::map<std::string, Power>& cards)
...
 
+    // ---------------------------------------- MINION - NEUTRAL
+    // [EX1_095] Gadgetzan Auctioneer - COST:5 [ATK:4/HP:4]
+    // - Faction: Neutral, Set: Expert1, Rarity: Rare
+    // --------------------------------------------------------
+    // Text: Whenever you cast a spell, draw a card.
+    // --------------------------------------------------------
+    power.ClearData();
+    power.AddTrigger(Trigger(TriggerType::CAST_SPELL));
+    power.GetTrigger().value().triggerSource =
```
Comment (utilForever) [inline Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: `triggerSource` is not `MINIONS_EXCEPT_SELF` but `FRIENDLY`.

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp (hunk 1 above)
Comment (thrk941106) [reply Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: Thank you! i'll fix it

Comment (utilForever) [reply Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: I confirmed it. Resolved.

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp (hunk 2 above)
Comment (utilForever) [reply Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: I confirmed it. Resolved.

**Pull request merged**